content/documentation/getting-started/index.adoc

```diff
@@ -196,6 +208,18 @@ bash <(curl -L https://git.io/getLatestKialiOperator) --uninstall-mode true
 ----
 
 
+=== Known Problem: Uninstall Hangs
+
+In Kiali < 0.21 and Kubernetes versions < 0.14 there is an operator-sdk bug that can hang uninstall.  This can happen when uninstalling Kiali via the Kiali Operator, or occasionally when trying to delete the namespace in which Kiali is installed.  This is because the Kiali Operator failed to successfully run its finalizer. If you get into this hung state the following command may resolve the problem:
```

Comment:
`because the Kiali Operator failed to successfully run its finalizer`

This isn't true. It does run the finalizer. k8s just has a bug in it that causes it to fail to empty the finalizer list. And this bug still exists if on OS 3.11 or K8s 1.11. It is just that we attempt to workaround it now in the deploy script. and that hasn't been released yet. So this is still a potential now.

Reply:
I took that text from your previous blob. I'll fix it though.